SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestFlowController.cpp

```diff
@@ -171,6 +175,14 @@ const FString ASpatialFunctionalTestFlowController::GetDisplayName()
 void ASpatialFunctionalTestFlowController::OnTestFinished()
 {
 	StopStepInternal();
+	if (WorkerDefinition.Type == ESpatialFunctionalTestWorkerType::Server)
+	{
+		bHasAckFinishedTest = true;
```

Comment:
What if we don't have authority over this particular controller? Or can that not happen?

Reply:
Michael also brought this up as it is confusing. In this situation, when this function is called this will always be equivalent to checking HasAuthority().
The reason why I'm not using that was because in the past I was told to not use it whenever I can just use the worker definition. Since in this situation there's no clear context that if it gets here are it's the server that you have authority I think I'll just replace it with HasAuthority().